erpnext/compliance/doctype/package_tag/package_tag.json

```diff
@@ -87,9 +88,16 @@
    "fieldtype": "Float",
    "label": "Package Tag Quantity",
    "read_only": 1
+  },
+  {
+   "default": "0",
+   "fieldname": "is_used",
+   "fieldtype": "Check",
+   "label": "Is used",
+   "read_only": 1
```

Comment:
@hrwX Should we allow users to manually change this value? In case they misplace the tag or destroy it and want to mark it as used.

Reply:
@neilLasrado the option can be secured using Perm Level so not everyone is able to edit it?

Reply:
@neilLasrado Have added a button to set tag as lost/destroyed